### ISSUE TITLE
Support for JMS\Serializer >= 2.0.0

### ISSUE THREAD

**Issue by tburschka** [allure-framework/allure-php-commons#41]
Currently i'm getting the error
```
[Semantical Error] The annotation "@JMS\Serializer\Annotation\XmlNamespace" in class Yandex\Allure\Adapter\Model\TestSuite does not exist, or could not be auto-loaded. 
```

Since 2.0.0 of the JMS\Serializer the path does not match anymore the autoloading structure in the AnnotationProvider:

current
```
/my-project/vendor/allure-framework/allure-php-api/src/Yandex/Allure/Adapter/Annotation/../../../../../../../../vendor/jms/serializer/src/JMS/Serializer/Annotation/XmlNamespace.php
```

expected
```
/my-project/vendor/allure-framework/allure-php-api/src/Yandex/Allure/Adapter/Annotation/../../../../../../../../vendor/jms/serializer/src/Annotation/XmlNamespace.php
```

**Comment by bmd** [comment]
Is there any chance of @SnomeaD 's pull request getting merged here? This s a big pain point for running alongside libraries that require the newer version of the serializer.

**Comment by m-tyrala** [comment]
Hello @tburschka, have you (or anyone) done something with these changes?  Any forked repo with new dependencies? I'd be grateful for any info, if it's abandoned, then I have to work on my own forks.

**Comment by tburschka** [comment]
Nope, i have no fork, since i've hoped for support here...

**Comment by remorhaz** [comment]
It's worth noting that old `jms\serializer` emits deprecations under PHP 8.0, so ping. I've resolved conflicts of #54 in #65.
